### PR TITLE
Update API usage in language intro tutorials

### PR DESCRIPTION
**Pull request by eb8680**
Addresses the language intro portion of #930 

This PR updates the language intro tutorials to be compatible with the API introduced in #1019 by using `TracePosterior.run` and `EmpiricalMarginal` instead of `Marginal`.

### CODE REVIEW

File: tutorial/source/intro_part_i.ipynb [hunk 1]
```diff
@@ -3,7 +3,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": true
+   },
```
Comment (fehiepsi) [inline tutorial/source/intro_part_i.ipynb]: Nit: you can remove these metadata by using `make scrub` :)

File: pyro/infer/svi.py [hunk 2]
```diff
@@ -51,15 +51,6 @@ def _loss_and_grads(*args, **kwargs):
             self.loss = loss
             self.loss_and_grads = loss_and_grads
 
-    def __call__(self, *args, **kwargs):
```
Comment (martinjankowiak) [inline pyro/infer/svi.py]: why is this being removed here?

Comment (eb8680) [reply pyro/infer/svi.py]: We don't want to support this API, and I remembered it while going over the tutorial

File: tutorial/source/intro_part_ii.ipynb [hunk 3]
```diff
@@ -346,7 +360,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": true
+   },
```
[the remark this answers is not in the document]
Comment (eb8680) [reply tutorial/source/intro_part_ii.ipynb]: It's all runnable, but none of these cells take more than a couple seconds so I think it's fine

File: tutorial/source/intro_part_i.ipynb [hunk 4]
```diff
@@ -163,12 +161,12 @@
     "    if t is None:\n",
     "        t = 0\n",
     "    x = pyro.sample(\"x_{}\".format(t), dist.Bernoulli(p))\n",
-    "    if torch.equal(x.data, torch.zeros(1)):\n",
+    "    if x.item() == 0:\n",
```
Comment (fritzo) [inline tutorial/source/intro_part_i.ipynb]: nit: actually `if x == 0:` now works even without `.item()`

**Pull request merged**